cli-tools/src/main/java/com/radixdlt/cli/GenerateGenesis.java

```diff
@@ -180,14 +183,45 @@ public static void main(String[] args) throws Exception {
         Base64.getEncoder().encodeToString(compressedGenesisData);
 
     if (validatorsCount > 0) {
+      // For some reason we use the presence of the validatorsCount command to decide whether we are
+      // running this for docker, or for an actual environment...
       System.out.format("export RADIXDLT_GENESIS_DATA=%s%n", compressedGenesisDataBase64);
     } else {
-      try (var writer = new BufferedWriter(new FileWriter("genesis.base64"))) {
-        writer.write(compressedGenesisDataBase64);
+      System.out.format(
+          "The base64-encoded genesis for use with RADIXDLT_GENESIS_DATA / network.genesis_data"
+              + " is:%n%s%n",
+          compressedGenesisDataBase64);
+      System.out.println();
+      var filePath = new File("genesis_data.bin").getAbsolutePath();
+      System.out.format(
+          "Also saving the genesis data file in binary format for use with"
+              + " RADIXDLT_GENESIS_DATA_FILE / network.genesis_data_file to: %s%n",
+          filePath);
+      try (var outputStream = new java.io.FileOutputStream(filePath)) {
+        outputStream.write(compressedGenesisData);
       }
     }
   }
 
+  private static Network parseNetwork(String commandLineValue) {
+    final var asName = Network.ofName(commandLineValue);
+    if (asName.isPresent()) {
+      return asName.get();
+    }
+    try {
+      final var asId = Network.ofId(Integer.parseInt(commandLineValue));
```

Comment:
if the `Integer.parseInt` is the reason for the weird `throw/catch`, then may I recommend `Ints.tryParse()`?